script/helperFunctions.sh

```diff
@@ -722,7 +722,7 @@ function saveDiamondPeriphery_MULTICALL_NOT_IN_USE() {
   while [ $attempts -lt 11 ]; do
     echo "Trying to execute multicall now - attempt ${attempts}"
     # try to execute call
-    MULTICALL_RESULTS=$(cast send "$MULTICALL_ADDRESS" "aggregate((address,bytes)[]) returns (uint256,bytes[])" "$MULTICALL_DATA" --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --rpc-url "https://polygon-rpc.com" --legacy)
+    MULTICALL_RESULTS=$(cast send "$MULTICALL_ADDRESS" "aggregate((address,bytes)[]) returns (uint256,bytes[])" "$MULTICALL_DATA" --account $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --rpc-url "https://polygon-rpc.com" --legacy)
```

Comment:
I dont really understand these changes. 
I would have expected that instead of `--private-key 4136723423....` we would now use `--account <KeyName>`  but the helper function `getPrivateKey` is not adapted by this PR so it would still return a private key, wouldnt it?

Reply:
I just did a bunch of find/replace so overlooked this. Will fix. Thanks!